src/utils/questionnaireBuilderUtils.ts

```diff
@@ -193,19 +177,27 @@ export const countCorrectAnswers = (questionnaire: Questionnaire): number => {
  * @returns Tooltip message
  */
 export const getTooltipMessage = (questionnaire: Questionnaire): string => {
-  const isTitleEmpty = !questionnaire.title.trim();
-  const isDescriptionEmpty = !questionnaire.description.trim();
+  if (!questionnaire.title || !questionnaire.description) {
```

Comment:
Seems like you already have this function here I suggested creating to do the validation for the save. So again, the way I think to best handle this is to disable the button and give the tool tips. I don't think we need "double validation" in the UI. Also the lamdba should have its own validations for the Questionnaire body it receives so we don't need to worry about that.